pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerMeter.java

```diff
@@ -102,7 +107,23 @@ public enum BrokerMeter implements AbstractMetrics.Meter {
   NETTY_CONNECTION_BYTES_RECEIVED("nettyConnection", true),
 
   PROACTIVE_CLUSTER_CHANGE_CHECK("proactiveClusterChangeCheck", true),
-  DIRECT_MEMORY_OOM("directMemoryOOMCount", true);
+  DIRECT_MEMORY_OOM("directMemoryOOMCount", true),
+
+
+  /**
+   * Number of multi-stage queries that have been started.
+   * <p>
+   * Unlike {@link #MULTI_STAGE_QUERIES_BY_TABLE}, this metric is global and not attached to a particular table.
+   * That means it can be used to know how many multi-stage queries have been started in total.
+   */
+  MULTI_STAGE_QUERIES_EXECUTED("queries", true),
```

Comment:
Suggest renaming them to `MULTI_STAGE_QUERIES_GLOBAL` and `MULTI_STAGE_QUERIES`

Reply:
Also way may put them next to `QUERIES`

Reply:
> Also way may put them next to QUERIES

In general I try to avoid to change the order of literals to make sure backward compatibility is not broken, but I think in this case it is fine to do it.

Changing it...